test/measures/measures-data-spec.js

```diff
@@ -21,6 +21,40 @@ describe('measures data json', function() {
         .filter(x => requiredAttestationIdsSet.has(x)));
       assert.equal(intersection.size, requiredAttestationIdsSet.size);
     });
+    it('should not have substitutes', () => {
```

Comment:
newline before this test case, and we prefer writing testcases without 'should' - so `it('does not have substitutes')` is just as legible